frontend/src/components/place/SearchPlace.vue

```diff
@@ -170,7 +170,7 @@ export default {
         updateMap() {
             if (this.isMapLoaded && !this.isLoading) {
                 this.markerManager.setMarkersFromPlacesAndFit(...this.places);
-                if (this.$route.query.location && this.places.length > 0) {
+                if (this.$route.query.location && this.places.length === 0) {
```

Comment:
what would be if this.places > 0, map isn't scaled? 
maybe we can avoid `this.places.length === 0`

Reply:
map will be scaled to fit all markers